CHANGELOG.md

```diff
@@ -5,10 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/global-121/121-platform/compare/v1.120.0...main)
+## [Unreleased](https://github.com/global-121/121-platform/compare/v1.130.0...main)
 
 ---
 
+## [1.130.0](https://github.com/global-121/121-platform/compare/v1.120.0...v1.130.0)- 2024-06-05
+
+### Added
+
+- [AB#28422](https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/28422) Show completed status on people affected page and default sort as registrationCreated for PAs
+
+### Technical
```

Comment:
I added this for our own internal "FYI" - but I wouldn't add this to an external stakeholder message